features/FEATURE_BOOTLOADER/targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC55S69/mbed_lib.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "bootloader_LPC55S69",
+    "target_overrides": {
+        "*": {
+            "target.app_offset": "0x8400",
+            "target.header_offset": "0x8000",
+            "target.bootloader_img": "mbed-bootloader-lpc55s69-v1_0_0.hex"
```

Comment:
What does this bootloader do?

Reply:
Please see https://os.mbed.com/docs/mbed-os/v5.11/tutorials/bootloader.html (Prebuilt bootloaders section)